tools/djxl_ng_main.cc

```diff
@@ -233,6 +235,107 @@ int DecompressJxlReconstructJPEG(const std::vector<uint8_t>& compressed,
   return EXIT_SUCCESS;
 }
 
+int DecompressJxlToPackedPixelFile(const std::vector<uint8_t>& compressed,
+                                   jxl::extras::PackedPixelFile& pixels,
```

Comment:
The name `pixels` looks a bit misleading here. How about renaming this to `ppf`?